utils/lib/rr_graph_fb/flatbuffers.py

```diff
@@ -0,0 +1,26 @@
+""" These functions should be moved to the flatbuffers.py library itself."""
```

Comment:
Remove the `rr_graph_fb` from this patch. Lets land that in a different one.

Reply:
Done